includes/custom_op_lite.h

```diff
@@ -418,6 +418,21 @@ struct Variadic : public TensorBase {
   TensorPtrs tensors_;
 };
 
+#ifdef USE_CUDA
+
+struct CudaContext {
+  void Init(const OrtW::CustomOpApi& api, const OrtKernelContext& ctx) {
+    const auto& ort_api = api.GetOrtApi();
+    ort_api.KernelContext_GetResource(&ctx, 1, 10000, &cuda_stream);
```

Comment:
what's 10000?

Reply:
Added an enum to specify the resource type.